central/config/datastore/singleton.go

```diff
@@ -55,6 +56,34 @@ var (
 		},
 		ExpiredVulnReqRetentionDurationDays: DefaultExpiredVulnReqRetention,
 	}
+
+	defaultVulnerabilityDeferralConfig = &storage.VulnerabilityDeferralConfig{
+		ExpiryOptions: &storage.VulnerabilityDeferralConfig_ExpiryOptions{
+			DayOptions: []*storage.DayOption{
+				{
+					NumDays: 14,
```

Comment:
nit:  Do with this what you will.  For consistency should these be constants like above?

Note, I say that, but the way you have it is probably more readable, it just depends on how strict we want to be on consistency within the file.

Reply:
Representing the information this way is more intuitive. Hence, I will keep it this way. The number of options is not constant. Since the time we started discussing this feature, the option count increased. 

Separately, the above default constants should not be exported.